app/validators/income_benefits_validator.rb

```diff
@@ -5,23 +5,19 @@ def validate(record)
     @record = record
 
     record.types.each_with_index do |type, index|
-      next if type == IncomeBenefitType::NONE.to_s
+      next if type == 'none'
 
       income_benefit = record.public_send(type)
       add_indexed_errors(income_benefit, index) unless income_benefit.valid?
     end
 
-    require_type?
+    return unless record.types.empty?
```

Comment:
Would this be better to go before the loop?

Reply:
I have fixed this in the task list.